client-app/src/desktop/tabs/other/routing/SimpleRoutingPanel.tsx

```diff
@@ -0,0 +1,98 @@
+import {HoistModel, hoistCmp, creates, XH} from '@xh/hoist/core';
+import {grid, GridModel} from '@xh/hoist/cmp/grid';
+import {StoreRecordId} from '@xh/hoist/data';
+import {Icon} from '@xh/hoist/icon';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {wrapper} from '../../../common';
+import React from 'react';
+
+export const simpleRoutingPanel = hoistCmp.factory({
+    displayName: 'SimpleRoutingPanel',
+    model: creates(() => new SimpleRoutingPanelModel()),
+
+    render({model}) {
+        return wrapper({
+            description: [
+                <p>
+                    Hoist provides functionality for route parameters to interact with UI
+                    components. Below is a simple grid whose selected record can be maintained by a
+                    route parameter.
+                </p>,
+                <p>
+                    E.g. URLs like <code>https://toolbox.xh.io/app/other/simpleRouting/123</code>,
+                    where <code>123</code> is a record ID and that record is (auto) selected in the
+                    grid. Additionally, the route parameter will be updated when the user selects a
+                    different record in the grid.
+                </p>
+            ],
+            item: panel({
+                title: 'Simple Routing',
+                icon: Icon.gridPanel(),
+                className: 'tb-simple-routing-panel',
+                item: grid({model: model.gridModel}),
+                height: 600,
+                width: 600
+            })
+        });
+    }
+});
+
+class SimpleRoutingPanelModel extends HoistModel {
+    gridModel = new GridModel({
```

Comment:
This should be marked as `@managed` - that's important for any models created by a model, to ensure they are properly destroyed and their internal reactions disposed of to allow for proper GC if this component were unmounted.